OpenOversight/app/models/database.py

```diff
@@ -279,6 +285,14 @@ class Officer(BaseModel, TrackUpdates):
         CheckConstraint("gender in ('M', 'F', 'Other')", name="gender_options"),
     )
 
+    def __repr__(self):
+        if self.unique_internal_identifier:
+            return (
+                f"<Officer ID {self.id}: {self.full_name()} "
```

Comment:
Shouldn't this be...?
```suggestion
                f"<Officer ID: {self.id} : {self.full_name()} "
```

Reply:
Done!